app/src/main/java/org/schabi/newpipe/about/LicenseFragment.java

```diff
@@ -44,16 +47,21 @@ public static LicenseFragment newInstance(final SoftwareComponent[] softwareComp
      * @param context the context to use
      * @param license the license to show
      */
-    private static void showLicense(final Context context, final License license) {
-        new LicenseFragmentHelper((Activity) context).execute(license);
+    private static void showLicense(final Activity context, final License license) {
+        new LicenseFragmentHelper(context).execute(license);
     }
 
     @Override
     public void onCreate(@Nullable final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         softwareComponents = (SoftwareComponent[]) getArguments()
                 .getParcelableArray(ARG_COMPONENTS);
-
+        if (savedInstanceState != null) {
+            Serializable license = savedInstanceState.getSerializable(LICENSE_KEY);
```

Comment:
Add `final`

Reply:
Done

Reply:
@nmurali94: If you've fixed such an issue, next time please just press the "Resolve conversation" button instead of saying something like "Done".